containers/Investment/RevolvingPool/View/EpochOverview.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react'
+import { Box, Button, Heading, Table, TableBody, TableRow, TableCell } from 'grommet'
+import { ITinlake as ITinlakeV3 } from '@centrifuge/tinlake-js-v3'
+import { createTransaction, useTransactionState, TransactionProps } from '../../../../ducks/transactions'
+import { connect } from 'react-redux'
+
+interface Props extends TransactionProps {
+  tinlake: ITinlakeV3
+}
+
+const EpochOverview: React.FC<Props> = (props: Props) => {
+  const [epochId, setEpochId] = React.useState(0)
+  // const [epochMinimumTimeEnd, setEpochMinimumTimeEnd] = React.useState(0)
+  const [epochState, setEpochState] = React.useState('')
+
+  // const endHoursFromNow = Math.floor((epochMinimumTimeEnd - new Date().getTime()) / 1000 / 60 / 60)
```

Comment:
See other comment on using local `Date`.

Reply:
Yeah I think we'd have to implement a `getEpochEndHours` or something to tinlake.js for this.

Reply:
But that's not barebones MVP, I just tried adding it quickly, so let's leave it out.

Reply:
Okay, fine for me – could you add a ticket though?

Reply:
This is just part of the "Epoch Table", so that's already on our todo list.